### ISSUE TITLE
Endorse codelists

### ISSUE THREAD

**Issue by pvgenuchten** [soilwise-he/governance#2]
Codelists are best endorsed/mandated by an authority or vivid community

How will Soilwise interact with authorities/communities to have soil codelists adopted?

Current and upcoming codelist initiatives, which may need confirmation by a EU authority

- [Glosis codelists](https://github.com/glosis-ld/glosis/blob/master/csv_codelists/glosis_cl.csv)
- [IUSS WRB codelists](https://wrb.isric.org/)
- [INSPIRE codelists](https://inspire.ec.europa.eu/codelist/SoilProfileParameterNameValue)
- [Land management practices](https://qcat.wocat.net/en/wocat/)
- [Biology codelists](https://www.gbif.org/)
- [EEA GEMET](https://www.eionet.europa.eu/gemet/en/about/)
- [Agrovoc](https://agrovoc.fao.org/browse/agrovoc/en/page/c_89f35c33?clang=en)

New codelists on soil health indicators

A governance model for maintenance of codelists should be available for any endorsed codelist

Codelist repositories should be persistent and standardised (SKOS)

Are current codelists compatible with definitions in legislation?

**Comment by fennyvanegmond** [comment]
I'm assuming codelists in this issue equals vocabs. Since the observable properties and procedure descriptions are actually among the most relevant ones for applications

**Comment by fennyvanegmond** [comment]
I assume the soil health (legislation) and other legislation codelists and vocabs will be registered to INSPIRE at some point. Does anyone know who is responsible for that? DG DEGIS/a responsible person within each DG (ENV, AGRI, CLIMA, DEGIS, etc)? Or a JRC INSPIRE department?

**Comment by fennyvanegmond** [comment]
I'm working with the IUSS WRB and SIS WG's to try to arrange governance on the WRB codelists and possibly GloSIS and other soil related vocabs.  More news on that towards the end of May hopefully.

**Comment by pvgenuchten** [comment]
Each memberstate has a national contact point for inspire, this is the typical role to submit changes to the inspire registry, for European institutions, I’m not aware of the route.

**Comment by pvgenuchten** [comment]
Vocab and codelist are not equal, [vocabulary]( https://www.thesaurus.com/browse/vocabulary ) or thesaurus or glossary are a list of terms with their meaning. A code list or enumeration lists potential values of a property. Third similar type is the ontology or data model, which defines types and relations between types and can include enumerations for type properties

- [glosis ld](https://github.com/glosis-ld/glosis) is an ontology
- Wrb is a set of codelists (soil types, classes for observed soil properties)
- Inspire soil is a data model, including references to inspire registry (code lists)
- [agrivoc](https://agrovoc.org) is a vocabulary including some definitions for items of the wrb code lists

**Comment by fennyvanegmond** [comment]
I know the difference between codelists and vocabs, but my point was that governance for both codelists and vocabs is needed and perhaps the vocabs are even more important. Many relevant soil properties are continuous, not class variables. 
So if we discuss governance of the codelists, we should include the vocabs that belong to the same ontologies and classification systems as well.

**Comment by pvgenuchten** [comment]
is this in scope of soilwise? or better to manage outside soilwise

preferably outside soilwise, but if not picked up... soilwise has a strong need to have this organised

**Comment by fennyvanegmond** [comment]
Indeed, SoilWise is an asking party and can be a bridge to other Mission Soil projects. Also SoilWise can be a technical facilitator and I would propose to leave it at that and have the governance arranged outside of SoilWise for now. Also to make it sustainable. I'll use ESP, INSII/GloSIS WG, ISRIC, Mission Soil Data Cluster for that is the plan currently.

**Comment by RaduGiurgiu** [comment]
@fennyvanegmond @pvgenuchten can we clsoe this issue then?

**Comment by lbtgnn68** [comment]
I just referenced https://www.gbif.org/ as Biology codelists, might also of interest: https://edaphobase.org/edaphobase-datafields/#datafields

**Comment by lbtgnn68** [comment]
About soil health, apart from SML which should be mandatory, among the many other I would reference https://www.biofunctool.com/

**Comment by pvgenuchten** [comment]
ok to close, this is picked up by [soilvoc](https://github.com/soilwise-he/soil-vocabs) component

seems conclusion is: **yes** soilwise (EUSO) requires a voc component
